addons/hr_timesheet/tests/test_timesheet.py

```diff
@@ -434,3 +434,50 @@ def test_subtask_log_timesheet(self):
         })
         self.assertEqual(self.task1.subtask_effective_hours, 8, 'Hours Spent on Sub-tasks should be 8 hours in Parent Task')
         self.task1.child_ids = [Command.clear()]
+
+    def test_log_timesheet_with_analytic_tags(self):
+        """ Test whether the analytic tag of the project or task is set on the timesheet.
+            Test Case:
```

Comment:
I prefer adding a new blank to separate the title of the comment and its body. 😇 
```suggestion
        """ Test whether the analytic tag of the project or task is set on the timesheet.
        
            Test Case:
```